quickwit/quickwit-cluster/src/cluster.rs

```diff
@@ -301,6 +301,24 @@ impl Cluster {
         tokio::time::sleep(GOSSIP_INTERVAL * 2).await;
     }
 
+    /// This exposes in chitchat some metric about the cpu usage of cooperative pipelines.
+    /// The metric are exposed as follows:
+    /// Key:        pipeline_metrics:<index_uid>:<source_id>
+    /// Value:      79‰,76MB/s
```

Comment:
Just so it's clear that we're dealing with thousands.
```suggestion
    /// Value:      179‰,76MB/s
```